### PR TITLE
The Edit Iteration page has 'Back to App' and 'Back to Iteration' buttons

### PR DESCRIPTION
**Pull request by kyriakipapadimitriou**
User Interface Fix: on the 'Edit Iteration' view page the user can go back to the app's page or the iteration's page. I also changed the submit button's color and size to be more appealing to users.

### CODE REVIEW

**Review by kimhoang73 — APPROVED** [review]
looks good! i like the "Back to App" idea since sometimes you don't know where you're going back to lol

**Review by julies642 — APPROVED** [review]
Nice changes, the new Back buttons look more user friendly and should cause less confusion.

File: app/views/iterations/edit.html.haml [hunk 1]
```diff
@@ -54,5 +54,7 @@
     %br
 
     %br
-    = submit_tag 'Save Changes'
+    = submit_tag 'Save Changes', :class => 'btn btn-success pull'
```
Comment (csungwon) [inline app/views/iterations/edit.html.haml]: can you explain what 'pull' class does??

Comment (kyriakipapadimitriou) [reply app/views/iterations/edit.html.haml]: most of the submit buttons in other view files had "pull" which I thought was necessary to request changes in the database. I ran the command without pull and it worked the same, so I deleted pull from the command. Thanks a lot for the suggestion!

**Review by csungwon — APPROVED** [review]
Looks great! The back button will definitely will provide a better user experience!

**Pull request merged**